styles.css

```diff
@@ -59,19 +59,31 @@ section {
     max-width: 150px;
 }
 
+.footer-nav-bar {
+    float: left;
```

Comment:
Possible redundant code for same solution

Reply:
ommitted float:left